docs/coding_guidelines.md

```diff
@@ -74,6 +74,32 @@ print("This is old style of formatting and should be avoided:")
 "My string %s" % variable
 ```
 
+* **Docstring**:
+* Use capital letters properly
+* Every docstring should describing easily what the function/class/etc. do
+* The description should be as follow:
+```python
+def my_example(arg1, arg2):
+    """
+    This function is for showing how a proper docstring should
+    look like
+
+    Args:
+        arg1 (<type of argument>): Description of what this argument is and if
+        the
```

Comment:
Should "the" be indented in 4 spaces with the rest following it?

Reply:
Thanks!